Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -21,36 +22,41 @@ For day to day and App Center, we use certificates and profiles from our Enterpr
 1. Add shared developer account, "John Appleseed", credentials from 1Password to your Xcode settings in `Preferences -> Accounts` menu. 
 2. Run the following command in the terminal: 
 
+- to download development certificates and profiles to run the app on device from Xcode:
+
 ```shell
- $ bundle exec fastlane match {{Type}} --team_id {{TeamID}} 
+ $ bundle exec fastlane match_development
 ```
 
-Following options can be used for the command above:
+- to download enterprise certificates and profiles to make enterprise build locally:
 
-| Code signing identities for | Type param| TeamID param    |
-|-----------------------------|-----------|-------------------
-|Day-to-day development       |development|Enterprise TeamID|
-|App Center                   |enterprise |Enterprise TeamID|
+```shell
+ $ bundle exec fastlane match_enterprise
+```
 
-(You can find the `TeamID` parameter by searching for `$team_ids` in `/fastlane/Appfile` in the main repo.)
+- to download AppStore certificates and profiles to make AppStore build locally:
+
+```shell
+ $ bundle exec fastlane match_appstore
+```
 
 3. On prompt for passphrase to decrypt the repository, enter the one from 1Password vault.
 
 To troubleshoot, refer to `fastlane/Matchfile`, `fastlane/Appfile` and [documentation](https://docs.fastlane.tools/actions/match/)
 
-### TestFlight and AppStore
-
-For TestFlight and AppStore, we use certificates and profiles from our App Store developer team
-
-If need arises to create a Testflight or App Store build on your machine, run the following command:
+Mentioned lanes will download certificates and profiles for all the apps (and their extensions) specificed in Appfile in the `app_config`. You can specify `readonly:false` option if you want the profiles to be regenerated. Not that it will regenerate all invalid profiles for all the apps.
```

Comment:
```suggestion
Mentioned lanes will download certificates and profiles for all the apps (and their extensions) specificed in Appfile in the `app_config`. You can specify `readonly:false` option if you want the profiles to be regenerated. Note that it will regenerate all invalid profiles for all the apps.
```